Configure `Dominant Color` and `Fetchpriority` modules for their standalone plugins (#704)
WordPress/performance: modules/images/dominant-color/readme.txt

=== Dominant Color Images ===

Contributors:      wordpressdotorg
Requires at least: 6.1
Tested up to:      6.2
Requires PHP:      5.6
Stable tag:        1.0.0
License:           GPLv2 or later
License URI:       https://www.gnu.org/licenses/gpl-2.0.html
Tags:              performance, images, dominant color

> Adds support to store dominant color for an image and create a placeholder background with that color.

Reviewer: ```suggestion
Adds support to store the dominant color of an image and create a placeholder background of that color.
```

Author: Thanks @10upsimon! I'll update this throughout as this is also the module description that should therefore be updated the same way.

Author: Covered by #708.

== Description ==

> This plugin determines and stores the dominant color for media uploads within WordPress and then uses it to create a placeholder background with that color in the frontend while the image is loading.

Reviewer: ```suggestion
This plugin determines and stores the dominant color for images uploaded via the media library within WordPress and then uses it to create a placeholder background of that color in the frontend, visible until the image is loaded.
```

Nit pick comment, mostly just to have things read a tad smoother. I also suggested that we explicitly say _images_ as opposed to _media_ uploaded, as this does not apply to other media such as PDF etc, correct?

Author: Thanks, updating this now. I'll include the "newly uploaded" bit here too, similar to how we are changing the module description, to clarify that it only applies to those images.
